### ISSUE TITLE
There is no indication that ComboBox with IsEditable=False has keyboard focus

### ISSUE THREAD

**Issue by lasgor** [HandyOrg/HandyControl#188]
**Describe the bug**
Editable ComboBox has blue outline when it has keyboard focus. ComboBox which has editable set to false does not have blue outline, which is to be expected. But it also does not have focus rectangle, so there is no way of telling that it has keyboard focus.

**Expected behavior**
Focus rectangle for ComboBox when IsEditable=False.

**Environment**
 - .net [4.0]
 - IDE [vs2019]
 - Version [2.2.0]
